src/consts.ts

```diff
@@ -19,10 +19,40 @@ const is = (...x) => 音韻地位.屬於(...x);
 /** @type { 音韻地位["判斷"] } */
 const when = (...x) => 音韻地位.判斷(...x);
 
-if (!音韻地位) return [
-  // 在此輸入方案選項……
+if (!音韻地位) return [  // 在此輸入方案選項；以下提供一些範本，方案設定格式：https://github.com/nk2028/tshet-uinh-deriver-tools/wiki/%E6%8E%A8%E5%B0%8E%E6%96%B9%E6%A1%88%E8%A8%AD%E5%AE%9A%E9%A0%85%E5%88%97%E8%A1%A8%E6%A0%BC%E5%BC%8F
```

Comment:
```suggestion
if (!音韻地位) return [
  // 在此輸入方案選項；以下提供一些範本，方案設定格式：https://github.com/nk2028/tshet-uinh-deriver-tools/wiki/%E6%8E%A8%E5%B0%8E%E6%96%B9%E6%A1%88%E8%A8%AD%E5%AE%9A%E9%A0%85%E5%88%97%E8%A1%A8%E6%A0%BC%E5%BC%8F
```

這麼長的註解可以單開一行了😂

Reply:
```suggestion
if (!音韻地位) return [
  // 在此輸入方案選項；以下提供一些範例，方案設定格式見：https://github.com/nk2028/tshet-uinh-deriver-tools/wiki/%E6%8E%A8%E5%B0%8E%E6%96%B9%E6%A1%88%E8%A8%AD%E5%AE%9A%E9%A0%85%E5%88%97%E8%A1%A8%E6%A0%BC%E5%BC%8F
```
整個方案本身已經是一個範本（template）所以這裏我會偏好用範例（example）